kiosk_mcp/mcp_server.py

```
@@ -0,0 +1,251 @@
+    result = json.dumps(await complete_session(_spring, session_id), ensure_ascii=False)
+    await save_tool_log(_spring, session_id, "complete_session", "{}", result)
+    return result
+
+
+if __name__ == "__main__":
+    transport = os.getenv("MCP_TRANSPORT", "sse")
+    if transport == "stdio":
+        mcp_app.run()  # Claude Desktop 연결용
+    else:
+        port = int(os.getenv("MCP_SERVER_PORT", "8090"))
+        mcp_app.run(transport="sse", host="0.0.0.0", port=port)  # FastAPI 연결용
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)